<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Soundplow</class>
 <widget class="QMainWindow" name="Soundplow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soundplow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Soundplow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_search">
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Search for any song!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Suggested Format: [artist] - [song title]</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textbox_search">
            <property name="toolTip">
             <string>The same thing as searching on the Soundcloud website!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_search">
            <property name="toolTip">
             <string>Pressing this will display a dialog with the top 5 search results.</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_like">
       <attribute name="title">
        <string>Like</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enter your Soundcloud username!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Whenever you like a song, we'll download it for you instantly.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textbox_like">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>The Soundcloud username we will monitor for new liked songs.</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_like">
            <property name="toolTip">
             <string>Click to toggle listening for likes.</string>
            </property>
            <property name="text">
             <string>Start Listening</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_link">
       <attribute name="title">
        <string>Link</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enter a track link!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="text">
             <string>We'll keep track of them all and download them when you're ready.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textbox_link">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Enter a Soundcloud track link and then press &quot;Add To List&quot; below!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_link">
            <property name="toolTip">
             <string>Click to add to the list of tracks below.</string>
            </property>
            <property name="text">
             <string>Add To List</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="font">
             <font>
              <family>Nirmala UI Semilight</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QListWidget" name="track_list">
            <property name="toolTip">
             <string>These are all the tracks that have been entered so far and that will be downloaded if the button below is clicked.</string>
            </property>
            <property name="statusTip">
             <string>Press the Delete key to remove the currently selected track from the list.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_download_all">
            <property name="toolTip">
             <string>Click to download all the tracks added to the list above.</string>
            </property>
            <property name="text">
             <string>Download All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Output folder:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textbox_output">
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>The folder where downloaded songs will output to.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="log">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>A console log to see any progress updates, warnings, or errors that occur.</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textbox_search</sender>
   <signal>returnPressed()</signal>
   <receiver>button_search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textbox_like</sender>
   <signal>returnPressed()</signal>
   <receiver>button_like</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textbox_link</sender>
   <signal>returnPressed()</signal>
   <receiver>button_link</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
